### DISCUSSION TITLE
Having a docker-compose.yaml file to run/build both frontend and backend Dockerrun files in hot reload without the need for any manual commands intervention.

### DISCUSSION THREAD

**Discussion by ghost** [spinspire/pocketbase-sveltekit-starter#10] [Ideas]
Trying to create a docker-compose.yaml file to run/build both frontend and backend Dockerrun files in hot reload without the need for any manual commands intervention.

Kindly let me know if can help out :)

**Comment by jkdoshi** [comment]
Good news. I have added docker-compose.yml and other docker files to the repo. Please check it out and let me know how it goes for you.
